pom.xml

```diff
@@ -749,7 +755,7 @@
         <groupId>io.sundr</groupId>
         <artifactId>sundr-maven-plugin</artifactId>
         <version>${sundrio.version}</version>
-        <dependencies>
+        <!--dependencies>
```

Comment:
Do we need to keep that code commented out? Can't we just remove it?

Reply:
+1